frontend-app/src/nikhil_ui_task/student/StudentTable.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {Table} from 'antd';
+
+const StudentTable = (props) => {
+
+const {StudentList} =props;
+
+const columns = [
+    {
+        title : "Full Name",
+        dataIndex : "name",
+        key : "rollno",
+        fixed : "left",
+    },
+    {
+        title : "Roll Number",
+        dataIndex : "rollno",
+        key : "rollno",
+    },
+    {
+        title : "Class",
+        dataIndex : "xclass",
+        key : "rollno"
+    },
+    {
+        title : "Email",
+        dataIndex : "email",
+        key : "rollno",
+    },
+    {
+        title : "Contact Number",
+        dataIndex : "phone",
+        key : "rollno"
+    },
+    {
+        title : "Percentage",
+        dataIndex : "percentage",
+        key : "rollno"
+    },
+]
+  return (
```

Comment:
Pagination Bug:
first it shows 3 options, but on selecting on of the option it shows only 2 options